packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/display.js';
+
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorNeutralShadowAmbient,
+  colorNeutralShadowKey,
+  fontFamilyBase,
+  fontSizeBase200,
+  lineHeightBase200,
+  spacingHorizontalXS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+/**
+ * Styles for the tooltip component
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host(:not(:popover-open)) {
+    display: none;
+  }
+
+  :host {
+    --blockOffset: ${spacingVerticalXS};
+    --inlineOffset: ${spacingHorizontalXS};
+    background: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    border: 1px solid var(--colorTransparentStroke);
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    display: inline-flex;
+    filter: drop-shadow(0 0 2px ${colorNeutralShadowAmbient}) drop-shadow(0 4px 8px ${colorNeutralShadowKey});
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    inset-area: block-start;
+    line-height: ${lineHeightBase200};
+    margin: 0;
+    max-width: 240px;
+    padding: 4px 11px 6px;
+    position-try-fallbacks: flip-block flip-inline;
+    position: absolute;
+    width: max-content;
+    z-index: 1;
+  }
+
+  :host(:is([positioning^='above'], [positioning^='below'])) {
+    margin-block: var(--blockOffset);
+  }
+  :host(:is([positioning^='before'], [positioning^='after'])) {
+    margin-inline: var(--inlineOffset);
+  }
+
+  :host([positioning='above-start']) {
+    inset-area: block-start span-inline-end;
+  }
+  :host([positioning='above']) {
+    inset-area: block-start;
```

Comment:
Can we use `TooltipPositioning.above` here and similarly in other places where we set the `inset-area`? It might help us avoid mistakes if we ever need to make some changes.